app/helpers/requests_helper.rb

```diff
@@ -100,8 +100,9 @@ def format_date(date)
     Time.zone.parse(date.to_s).strftime('%Y-%m-%d %I:%M%P')
   end
 
-  def searchworks_link(item_id, item_title)
-    link_to item_title, "#{Settings.searchworks_link}/#{item_id}", 'data-behavior': 'truncate'
+  def searchworks_link(item_id, item_title, html_options = {})
+    html_options['data-behavior'] = 'truncate'
+    link_to item_title, "#{Settings.searchworks_link}/#{item_id}", html_options
```

Comment:
Do you think it would be worth adding a spec to `requests_helper_spec.rb` to validate that our HTML options get passed? I'm on the fence on if it would be useful or not, but it should be an easy enough spec to write.  Totally up to you.